mrmustard/lab_dev/transformations/amplifier.py

```diff
@@ -41,7 +41,7 @@ class Amplifier(Channel):
         >>> from mrmustard.lab_dev import Amplifier, Coherent
 
         >>> amp = Amplifier([0], gain=4)
-        >>> coh = Coherent([0], x=1.0, y=2.0)  # units of hbar=2 (default)
+        >>> coh = Coherent([0], x=1.0 * np.sqrt(2), y=2.0 * np.sqrt(2))  # units of hbar=1 (default)
```

Comment:
Thinking about user convenience should we try to make x and y hbar independent?

Reply:
The Bargmann representation is hbar-independent but coordinates in phase space are hbar-dependent by definition. I either had to put a sqrt(2) here or change the allclose statement below where I check the value of mu (which is hbar dependent). I'll change the value below to make it clear.